XenAdmin/Wizards/PatchingWizard/PatchingWizard_ModePage.cs

```diff
@@ -165,6 +185,29 @@ private void ManualRadioButton_CheckedChanged(object sender, EventArgs e)
             UpdateEnablement();
         }
 
+        private bool AnyPoolForbidsAutoRestart()
+        {
+            var poolForbidRestartKey = "XenCenter.CustomFields.hci-forbid-update-auto-restart";

```

Comment:
These keys are normally like "hci-forbid-update-auto-restart: 1;" without prefixes and usually just checking whether the key exists is enough, see for example the key rolling_upgrade_in_progress.
I would also suggest moving this to the pool extensions as well as avoiding the use of the Helpers.GetOtherConfig method on an object of known type, as it's not necessary.